cmake/std/atdm/ats1/custom_builds.sh

```diff
@@ -8,33 +8,21 @@
 # Custom compiler selection logic
 
 if atdm_match_any_buildname_keyword \
-     intel-19.0.4-mpich-7.7.6 \
-     intel-19.0.4_mpich-7.7.6 \
+     intel-19.0.4-mpich-7.7.15 \
+     intel-19.0.4_mpich-7.7.15 \
      intel-19.0.4 \
      intel-19 \
+     default \
```

Comment:
Need to add an 'intel' above this so that if user selects raw `intel` it will match this.